src/test/java/com/exasol/adapter/dialects/postgresql/PostgreSQLSqlDialectIT.java

```diff
@@ -1,622 +1,598 @@
+                + "myPoint POINT, " //
+                + "myPolygon POLYGON, " //
+                + "myReal REAL, " //
+                + "mySmallint SMALLINT, " //
+                + "myText TEXT, " //
+                + "myTime TIME, " //
+                + "myTimeWithTimeZone TIME WITH TIME ZONE, " //
+                + "myTimestamp TIMESTAMP, " //
+                + "myTimestampWithTimeZone TIMESTAMP WITH TIME ZONE, " //
+                + "myTsquery TSQUERY, " //
+                + "myTsvector TSVECTOR, " //
+                + "myUuid UUID, " //
+                + "myXml XML " //
+                + ")");
+        statementPostgres.execute("INSERT INTO " + qualifiedTableName + " VALUES (" //
+                + "10000000000, " // myBigint
+                + "nextval('" + qualifiedTableName + "_myBigserial_seq'::regclass), " // myBigserial
+                + "B'1', " // myBit
+                + "B'0', " // myBitVar
+                + "false, " // myBoolean
+                + "'( ( 1 , 8 ) , ( 4 , 16 ) )', " // myBox
+                + "E'\\\\000'::bytea, " // myBytea
+                + "'hajksdf', " // myCharacter
+                + "'hjkdhjgfh', " // myCharacterVar
+                + "'192.168.100.128/25'::cidr, " // myCidr
+                + "'( ( 1 , 5 ) , 3 )'::circle, " // myCircle
+                + "'2010-01-01', " // myDate
+                + "192189234.1723854, " // myDouble
+                + "'192.168.100.128'::inet, " // myInet
+                + "7189234, " // myInteger
+                + "INTERVAL '1' YEAR, " // myInterval
+                + "'{\"bar\": \"baz\", \"balance\": 7.77, \"active\": false}'::json, " // myJson
+                + "'{\"bar\": \"baz\", \"balance\": 7.77, \"active\": false}'::jsonb, " // myJsonB
+                + "'{ 1, 2, 3 }'::line, " // myLine
+                + "'[ ( 1 , 2 ) , ( 3 , 4 ) ]'::lseg, " // myLseg
+                + "'08:00:2b:01:02:03'::macaddr, " // myMacAddr
+                + "100.01, " // myMoney
+                + "24.23, " // myNumeric
+                + "'[ ( 1 , 2 ) , ( 3 , 4 ) ]'::path, " // myPath
+                + "'( 1 , 3 )'::point, " // myPoint
+                + "'( ( 1 , 2 ) , (2,4),(3,7) )'::polygon, " // myPolygon
+                + "10.12, " // myReal
+                + "100, " // mySmallint
+                + "'This cat is super cute', " // myText
+                + "'11:11:11', " // myTime
+                + "'11:11:11 +01:00', " // myTimeWithTimeZone
+                + "'2010-01-01 11:11:11', " // myTimestamp
+                + "'2010-01-01 11:11:11 +01:00', " // myTimestampwithtimezone
+                + "'fat & rat'::tsquery, " // myTsquery
+                + "to_tsvector('english', 'The Fat Rats'), " // myTsvector
+                + "'a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11'::uuid, " // myUuid
+                + "XMLPARSE (DOCUMENT '<?xml version=\"1.0\"?><book><title>Manual</title><chapter>...</chapter></book>') " // myXml
+                + ")");
```

Comment:
:( sorry...

Reply:
Not your fault. My admiration for the dedication of the PostgreSQL makers.
On the other hand they had 24 years since their first release to pile up this mass of supported types.